syntax = "proto3";
package auth;
option go_package = "api_with_milvus/pb";

message GetVectorRequest {
  string doc = 1;
}

message GetVectorResponse {
  repeated float docVector = 1;
}

service VectorManager {
  rpc GetVector (GetVectorRequest) returns (GetVectorResponse);
}